Prepare for beta16 (#826)
twilio/twilio-video.js: CHANGELOG.md

For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).

2.0.0-beta16 (December 10, 2019)
================================

New Features
------------

> - This release completes implementation of [Track Priority and Bandwidth Profiles API](https://www.twilio.com/docs/video/migrating-1x-2x#track-priority-and-bandwidth-profiles-group-rooms-only-private-beta) which gives you finer control over bandwidth usage by tracks

Reviewer: ```suggestion
- This release supports all the features of the [Track Priority and Bandwidth Profile APIs](https://www.twilio.com/docs/video/migrating-1x-2x#track-priority-and-bandwidth-profiles-group-rooms-only-private-beta).
```

> - You can now specify the mode to control track switch off behavior by specifying a

Reviewer: ```suggestion
- You can now specify the mode to control Track switch off behavior by specifying a
```

>   property `trackSwitchOffMode` in BandwidthProfile options. This can be set to one of

Reviewer: ```suggestion
  property `trackSwitchOffMode` in BandwidthProfileOptions. This can be set to one of
```

For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).

2.0.0-beta16 (December 10, 2019)
================================

New Features

> ------------

Reviewer: ```suggestion
 ------------

```

- This release supports all the features of the [Track Priority and Bandwidth Profile APIs](https://www.twilio.com/docs/video/migrating-1x-2x#track-priority-and-bandwidth-profiles-group-rooms-only-private-beta).

- You can now specify the mode to control Track switch off behavior by specifying a

>   property `trackSwitchOffMode` in BandwidthProfileOptions. This can be set to one of

Reviewer: @makarandp0 Please specify the JIRA ticket for this feature.

For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).

2.0.0-beta16 (December 10, 2019)
================================

New Features
 ------------

- This release supports all the features of the [Track Priority and Bandwidth Profile APIs](https://www.twilio.com/docs/video/migrating-1x-2x#track-priority-and-bandwidth-profiles-group-rooms-only-private-beta).

- You can now specify the mode to control Track switch off behavior by specifying a
  property `trackSwitchOffMode` in BandwidthProfileOptions (JSDK-2549).

>   `trackSwitchOffMode` can be set to one of

Reviewer: @makarandp0 Can you make sure the ticket is at the end the sentence for `disabled`?

- Fixed an issue where loading twilio-video.js in firefox with `media.peerconnection.enabled` set to false in `about:config` caused page errors. (JSDK-2591)

> Known Issues

Reviewer: @makarandp0 Please move this to COMMON_ISSUES.md and describe the exact scenario.

Author: @manjeshbhargav - I have moved this to COMMON_ISSUES.md and also added a common issue that we see on ios 13.0.1
